README.md

```diff
@@ -102,44 +102,26 @@ Validators update the miner every 20 minutes. It's normal not to observe request
 nc -vz [Your Miner IP] [Port]
 ```
 
```

Comment:
I'm against having different way of running the node for miner and validator. Either both should be with pm2 or make. And with make we need explicitly write that pm2 is used underneath. Plus, we need to say what file should be edited.